public/css/main.css

```
@@ -230,3 +230,11 @@ tr {
 .status-color-dot.none {
   background-color: #2ecc71;
 }
+
+#libLink {
+    text-decoration: none;
```

```suggestion
    text-decoration: none;
    display: inline-block;
```
Make this an inline block element to avoid the gap between the two lines not being a link.